### PR TITLE
[tools] (version-remover) Fixing brace parsing in version remover

### PR DESCRIPTION
**Pull request by AlexJerabek**
The version remover was incorrectly removing stuff when the inline links had ending curly braces right before a newline. This should fix that.

### CODE REVIEW

File: generate-docs/tools/version-remover.ts [hunk 1]
```diff
@@ -45,10 +52,10 @@ fsx.writeFileSync(process.argv[4], wholeDTS);
 
 
 function getDeclarationEnd(wholeDts: string, startIndex: number): number {
```
Comment (samantharamon) [inline generate-docs/tools/version-remover.ts:54]: Minor suggestion: For consistency in using the camelCase naming convention, consider also updating the `wholeDts` variable name declared in line 10.

**Review by samantharamon — APPROVED** [review]
Thanks for the fix. Approved with a minor suggsetion.

**Comment by AlexJerabek** [comment]
Thanks @samantharamon. Suggestion applied. I'll check this in once the build validates.

**Comment by ElizabethSamuel-MSFT** [comment]
Looks good

**Pull request merged**